app/views/posts/show.html.erb

```diff
@@ -15,7 +15,8 @@
  <li class="list-group-item">username:  <%= User.find(comment.user_id).name %> comment:  <%= comment.text%> </li> 
   <%end%>
   </ul>
-  <div class='d-flex justify-content-center align-items-center pt-5'>
-<button type="button" class="btn btn-primary">Pagnation</button>
+
+  <div class='d-flex justify-content-center align-items-center'>
+  <%= render 'likes/new', post: @post, like: @like %>
```

Comment:
- [ ] [Optional] In the posts index page, Please add a button to add a new post so the user can add a new post.
- [ ] [Optional] It was a requirement in a previous project that clicking on the user or the post redirects you to the user or post `show` page. So anyone can navigate your app easily. Please fix that.